conf.py

```diff
@@ -60,6 +60,7 @@
                 "stax2_api": "4.2.1",
                 "woodstox_core": "6.2.1",
                 "authorizer": "0.5.0",
+                "java-devel": "11",
```

Comment:
Nit: I think it would be great if we have the attribute `java-devel` right below `java-base`